lib/json_pure.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+require 'json/pure'
```

Comment:
This is a bit orthogonal, but ensure that adding `gem "json_pure"` in a Gemfile properly load `json/pure` in the expected order.